corehq/apps/userreports/models.py

```diff
@@ -495,6 +495,7 @@ class StaticDataSourceConfiguration(JsonObject):
     """
     _datasource_id_prefix = STATIC_PREFIX
     domains = ListProperty()
+    server_environment = ListProperty()
```

Comment:
(Minor), if we expect this to contain multiple environments, `server_environments` might be more suitable name.